kit/cli/viper.go

```diff
@@ -62,7 +62,7 @@ func NewCommand(v *viper.Viper, p *Program) *cobra.Command {
 
 	if configPath := v.GetString("CONFIG_PATH"); configPath != "" {
 		switch path.Ext(configPath) {
-		case ".json", ".toml", ".yaml", "yml":
+		case ".json", ".toml", ".yaml", ".yml":
```

Comment:
Is there any chance of running on Windows where we would need case-insensitive comparison?

Reply:
Case-insensitivity seems like a good idea either way, will update.